internal/operators/odf/validations.go

```diff
@@ -68,28 +70,31 @@ func (o *operator) computeResourcesAllNodes(cluster *models.Cluster, odfClusterR
 	var err error
 
 	hosts := cluster.Hosts
-	compactMode := int64(len(hosts)) == 3
+	isCompactMode := int64(len(hosts)) == 3
```

Comment:
Why `int64`?

Reply:
I don't think it has a solid reason, just left it as is

Reply:
Oh sorry, thought it was your diff